### PR TITLE
feat: Add fixtures

### PR DESCRIPTION
**Pull request by JF-Cozy**
use cozy-ach to import fixtures

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,11 @@
     "test": "cs test --verbose --coverage",
     "cozyPublish": "git fetch origin ${DEPLOY_BRANCH:-build}:${DEPLOY_BRANCH:-build} && cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})",
     "stack:docker:dev": "cs stackDocker",
-    "stack:docker:prod": "cs stackDocker --prod"
+    "stack:docker:prod": "cs stackDocker --prod",
+    "fixtures:drop": "ACH drop io.cozy.timeseries.geojson.aggregate io.cozy.timeseries.geojson",
+    "fixtures:accounts": "ACH import fixtures/accounts.json",
+    "fixtures:datas": "ACH script fixtures/importGeojson.js",
```
Comment (paultranvan) [inline package.json:23]: You could simply add cozy-ach in devDep and do `"fixtures:datas": "yarn run ACH script  timeseries/importGeojson"` and avoid copy/pasting the importGeojson.js ;)

File: fixtures/importGeojson.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    return [DOCTYPE_GEOJSON, DOCTYPE_GEOJSON_AGGREGATE]
+  },
+  run: async function(ach) {
+    const client = ach.client
+
+    console.log(`Import ${N_DOCS} trips...`)
+    const trips = await createTrips(client)
+
+    console.log(`Aggregate ${N_DOCS} trips...`)
+    await createAggregates(client, trips)
```
Comment (paultranvan) [inline fixtures/importGeojson.js:50]: We should probably add a param in the ACH script to make the aggregates optional, as we do not need it here

File: package.json (hunk 1 above)
Comment (paultranvan) [reply package.json:23]: By doing so, the  fixtures/geojson.json  file probably becomes not useful

**Comment by JF-Cozy** [comment]
le code a été déplacé dans cozy-ach : https://github.com/cozy/ACH/pull/113

**Review by paultranvan — COMMENTED** [review]
Looks good!

**Pull request merged**